.travis.yml

```diff
@@ -5,36 +5,39 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
 env:
-  - SYMFONY_VERSION=2.5.*
+  - SYMFONY_VERSION=2.7.*
 
 matrix:
   allow_failures:
-    - env: SYMFONY_VERSION=dev-master
-    - php: 5.6
     - php: hhvm
   include:
-    - php: 5.5
-      env: SYMFONY_VERSION=2.0.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.1.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.2.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.3.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
-    - php: 5.5
-      env: SYMFONY_VERSION=dev-master
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*@dev
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
 
-before_script:
+before_install:
+  - sh -c 'if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi;'
   - git config --global user.email "test@test.com"
   - git config --global user.name "John Doe"
   - composer self-update
-  - composer require symfony/console:${SYMFONY_VERSION} symfony/yaml:${SYMFONY_VERSION} symfony/process:${SYMFONY_VERSION} --no-update
-  - composer update symfony/console symfony/yaml symfony/process
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
```

Comment:
I've tried this setup but defined the components by hand (instead of symfony/symfony), but then the build fails with `Problem 1
    - Installation request for symfony/http-kernel 3.0.*@dev -> satisfiable by symfony/http-kernel[3.0.x-dev].
    - symfony/http-kernel 3.0.x-dev requires symfony/event-dispatcher ~2.8|~3.0 -> no matching package found.`.? See [build](https://travis-ci.org/acrobat/RecaptchaBundle/jobs/90475692)

So I guess you have to use symfony/symfony or you will have to change the minimum-stability to dev for the package